public/js/like.js

```diff
@@ -1,11 +1,11 @@
 
 $(document).ready(function() {
 
-	$('.like-btn').click(function () {
+	$(".like-btn").click(function () {
 
-		var
-		status 		= $(this).attr("like-status"),
-		like_count	= $(this).html();
+		var status 		  = $(this).attr("like-status"),
+		var like_count	  = $(this).html();
```

Comment:
Unexpected 'var'.<br>Identifier 'like_count' is not in camel case.<br>Expected an identifier and instead saw 'var' (a reserved word).<br>Missing semicolon.<br>'like_count' is not defined.<br>'var' is defined but never used.